lib/zendesk_apps_support/installed.rb

```diff
@@ -17,7 +17,8 @@ def compile(options = {})
         appsjs: @appsjs,
         installations: @installations,
         installation_orders: options.fetch(:installation_orders, {}),
-        rollbar_zaf_access_token: options.fetch(:rollbar_zaf_access_token, '')
+        rollbar_zaf_access_token: options.fetch(:rollbar_zaf_access_token, ''),
+        zaf_naughty_v1_logging: options.fetch(:zaf_naughty_v1_logging, '')
```

Comment:
I think our Arturos are normally prefixed with `apps_`. Should that be the case here?

Reply:
Yes, good catch